src/design-tokens/tier-3-component/buttons.json

```diff
@@ -128,7 +128,7 @@
                   "value": "{eds.theme.color.transparent.black.0}"
                 },
                 "hover": {
-                  "value": "{eds.color.neutral.600}"
+                  "value": "{eds.color.neutral.200}"
```

Comment:
This is the file I changed manually. All the other changes were generated automatically.